style.css

```diff
@@ -106,3 +110,202 @@ article {
 .social-media {
   margin-left: 26px;
 }
+
+.work-section {
+  margin: 420px 0 150px;
+  display: grid;
+  row-gap: 100px;
+  padding: 0 5%;
+  justify-items: center;
+}
+
+.work-card {
+  background: #fff;
+  border-radius: 10px;
+  min-height: 3em;
+  border: 1px solid #c6c6c6;
+  padding: 20px 15px;
+  display: grid;
+  margin-top: 70px;
+}
+
+.work_preview {
+  display: flex;
+  align-self: center;
+  flex-direction: column;
+  justify-self: center;
+  overflow: hidden;
+}
+
+.work_preview img {
+  transition: transform 0.2s ease-in-out;
+  max-width: 100%;
+}
+
+.work_preview:hover .project-img {
+  transform: scale(1.4);
+}
+
+.work_detail {
+  align-items: center;
+  padding: 10px;
+}
+
+.work_title {
+  margin: 10px 0;
+  color: #172b4d;
+  font-size: 32px;
+  font-weight: 800;
+  line-height: 44px;
+}
+
+.work_info {
+  display: flex;
+  overflow: hidden;
+  color: #7a869a;
+  padding-left: 1%;
+}
+
+.info_item {
+  margin-right: 15px;
+  list-style-position: inside;
+  margin-left: 1px;
+}
+
+.info_item:first-child {
+  font-weight: bold;
+  list-style: none;
+  color: #344563;
+}
+
+.info_item:last-child {
+  margin-right: 0;
+}
+
+.work_details_content {
+  margin: 10px 0;
+  padding: 8px 0;
+  color: #344563;
+}
+
+.btn {
+  cursor: pointer;
+  font-size: 18px;
+  margin-top: 20px;
+  color: #396df2;
+  padding: 10px 20px;
+  background: #fff;
+  border-radius: 10px;
+  display: inline-block;
+  border: 1px solid #396df2;
+  transition: color 0.5s ease-in-out, background 0.2s ease-in-out;
+}
+
+.btn:hover {
+  color: #fff;
+  background: #396df2;
+}
+
+.work_tech {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 10px 0;
+  padding-left: 1%;
+}
+
+.work_tech > li {
+  padding: 8px;
+  list-style: none;
+  color: #6070ff;
+  border-radius: 8px;
+  margin-right: 8px;
+  background: #ebebff;
+  font-size: 14px;
+  line-height: 16px;
+}
+
+.about {
+  display: grid;
```

Comment:
- [ ] As demanded by project requirements, make sure the about section is laid out using CSS flexbox.
![image](https://github.com/Afsheen96/Portfolio/assets/94551484/ae882829-67f3-4796-9f75-9da5d5cc2992)